tests/scripts/thread-cert/Cert_5_3_01_LinkLocal.py

```diff
@@ -36,6 +36,7 @@
 
 LEADER = 1
 DUT_ROUTER1 = 2
+DATA_LEN = 256
```

Comment:
nit: rename as `FRAGMENTAL_LEN` to be more self-descriptive?

Reply:
It is a ICMP payload length, not including the ICMP header, FRAGMENTAL_LEN is more like the whole ICMP packet length, thoughts?

Reply:
This variable is used in below ping() method as payload length configuration, with a sample value which would surely causes fragmentation over 15.4 link.

Not so clear to me how it would cause the confusion you mentioned. 

Fine to me if you prefer to keep it if you would add some comment for its usage.

Reply:
I see, change to FRAGMENTED_DATA_LEN